src/snowflake/connector/pandas_tools.py

```diff
@@ -362,6 +362,11 @@ def drop_object(name: str, object_type: str) -> None:
         logger.debug(f"dropping {object_type} with '{drop_sql}'")
         cursor.execute(drop_sql, _is_internal=True)
 
+    def truncate_table(name: str) -> None:
+        truncate_sql = f"TRUNCATE TABLE IF EXISTS {name} /* Python:snowflake.connector.pandas_tools.write_pandas() */"
```

Comment:
I believe that it'd be quite an unexpected situation if it didn't exist. 
```suggestion
        truncate_sql = f"TRUNCATE TABLE {name} /* Python:snowflake.connector.pandas_tools.write_pandas() */"
```